scripts/job_manager.ps1

```diff
@@ -0,0 +1,94 @@
+class MyJobManager {
+    [String] $Name = "JobManager"
+    [Hashtable] $Jobs = @{}
+
+    JobManager () {
+    }
+
+    [void] AddJob ($Tracker, $ScriptBlock, $ArgumentList, $InitScript) {
+        $job = Start-Job -ScriptBlock $ScriptBlock `
+                         -ArgumentList $ArgumentList `
+                         -InitializationScript $InitScript
+        Start-Sleep -Milliseconds 100
+        if ($job.State -ne "Running" ) {
```

Comment:
are there other states you need to keep track of? what are the possible states of a job?

Reply:
the job should be in the running state; if something bad happens, we would like to see the output and not add the job to the tracker regardless of the state the job is in.